test/e2eTests/pages/registerPatientPage.js

```diff
@@ -160,7 +160,7 @@ class RegisterPatientPage extends Page {
    */
   clickNext(seconds = 5) {
     this.I.click(this.buttons.nextStep);
-    this.I.waitForInvisible('#overlay', seconds);
+    this.I.waitForInvisible('#overlay');//, seconds);
```

Comment:
What about a spinny wheel that never hides? Will the test just run on forever or is there a timeout somewhere that will fail a test a fter a certain amount of time.

I love the removal of all of the time dependent code. This is great!!!

Reply:
The default timeout is used for all the waitFor functions when no timeout is provided, (except waitForNavigation), it is set in the codecept.conf.js, currently set to 30s.